Payments/UserInterface/03a_ListRefunds.md

```diff
@@ -0,0 +1,234 @@
+
+    - ![Points](../../Assets/Icons/Points01.png "[Points]") (Points)      
+        Click this button to the right of the *View* drop-down list to display the context menu. The following menu entries are available:
+
+        - ![Create](../../Assets/Icons/Plus06.png "[Create]") create  
+            Click this entry to create a view. The *Create view* window is displayed, see [Create view](#create-view).
+
+        - ![Rename](../../Assets/Icons/Edit02.png "[Rename]") rename  
+            Click this entry to rename the selected view. The *Rename view* window is displayed, see [Rename view](#rename-view). This menu entry is only displayed if a view has been selected.
+
+        <!---- Eintrag gibt es nicht mehr [Reset](../../Assets/Icons/Reset.png "[Reset]") reset  
+            Click this entry to reset all unsaved changes to the settings of the selected view. This menu entry is only displayed if a view has been selected and any changes have been made to the view settings.--->
+
+        - ![Publish](../../Assets/Icons/Publish.png "[Publish]") publish  
+            Click this entry to publish the view. This menu entry is only displayed if you have selected an unpublished view. As long as you have not published a view, your views are visible for you only.
+
+        - ![Unpublish](../../Assets/Icons/Unpublish.png "[Unpublish]") unpublish  
+            Click this entry to unpublish the view so that it is visible only to you again. This menu entry is only displayed if you have selected and published a view.
+
+        - ![Save](../../Assets/Icons/Save.png "[Save]") save  
+            Click this entry to save the current view settings in the selected view. This menu entry is only displayed if you have selected a view.
+
+            > [Info] When the settings of a view have been changed, an asterisk is displayed next to the view name. The asterisk is hidden as soon as the changes have been saved.
+
+        - ![Delete](../../Assets/Icons/Trash01.png "[Delete]") delete  
+            Click this entry to delete the selected view. A confirmation window to confirm the deletion is displayed. This menu entry is only displayed if you have selected a view.
+ 
+
+- *Connections*   
+    Click the drop-down list to select a connection. The following options are available:
+    - **All connections**  
+        Select this option to display authorization transactions for all configured connections. 
+    - **Connection name**  
+        Select this option to display authorization transactions for the selected connection. When a specific connection has been selected, the **All connections** option is no longer displayed. Click the selected connection again to deselect it.  
+
+
+
+The list displays all refund transactions. Depending on the payment service provider, the displayed columns vary. All fields are read-only. The following standard columns are available.
+<!----Liste der Standardfelder aus den Authorizations kopieren--->
```

Comment:
Änderungen in Authorizations hier auch vornehmen.

Reply:
ok